frontend/src/router/index.ts

```diff
@@ -12,12 +12,12 @@ const router = createRouter({
       redirect: '/home'
     },
     {
-      path: '/home',
-      component: HomeView
+      path: '/createrole',
+      component: CreateView
```

Comment:
The name of the component is still unclear. CreateView means what? Create what?

Besides, the pattern of path should be /create-role